### PR TITLE
Add more information to custom code docs

### PR DESCRIPTION
**Pull request by thomashacker**
## Description

This `PR` adds a small warning section to the `usage/training` docs that mentions that unpackaged models, trained with custom code, do not internally contain custom code, but need to be imported into the script before loading in the models. 

### Types of change

Documentation

## Checklist

- [x] I confirm that I have the right to submit this contribution under the project's MIT license.
- [x] I ran the tests, and all new and existing tests passed.
- [x] My changes don't require a change to the documentation, or if they do, I've added all required information.

### CODE REVIEW

File: website/docs/usage/training.mdx [hunk 1]
```diff
@@ -758,6 +758,14 @@ any custom architectures, functions or
 your pipeline and registered when it's loaded. See the documentation on
 [saving and loading pipelines](/usage/saving-loading#models-custom) for details.
 
+<Infobox variant="warning">
+
+Please note that unpackaged models produced by `spacy train` do **not include
+any custom code**, you need to import the code to your script before loading in
```
Comment (adrianeboyd) [inline website/docs/usage/training.mdx:764]:
```suggestion
any custom code**, you need to import the code in your script before loading
```

**Comment by adrianeboyd** [comment]
This is a good addition! I'm thinking a little about whether it should also include an explanation related to `spacy.load("/path")` vs. `spacy.load("package_name") in terms of whether code can be included/loaded. I wonder if we can link to a section with more details, but I'm not sure there is one in the docs?

**Comment by thomashacker** [comment]
We could link to this section https://spacy.io/api/top-level#spacy.load
It mentions that installed packages run their own `.load()` method. We could mention that this is where the custom code gets imported.

**Comment by rmitsch** [comment]
> We could link to this section https://spacy.io/api/top-level#spacy.load
It mentions that installed packages run their own .load() method. We could mention that this is where the custom code gets imported.

I think this doesn't cover the difference between `spacy.load("/path")` and `spacy.load("package_name")` that Adriane referred to. [This](https://spacy.io/usage/saving-loading#models) could also be linked and has some more info, but also doesn't explicitly discuss a difference. Maybe we should extend one of these two doc pages and then link them from the new doc section in this PR?  @adrianeboyd 

Otherwise LGTM.

**Comment by adrianeboyd** [comment]
I think it would make sense to expand the top paragraph at https://spacy.io/api/top-level#spacy.load to clarify that loading from a package name includes loading any code packaged with it, but loading from data directories only loads data and no additional code?

And then link that?

**Comment by adrianeboyd** [comment]
Ah, this should also be switched to `master`. I'll switch it here, but it will probably need to be redone not to include unintended commits (force-pushing to fix it is fine).

**Pull request closed without merging**